newsroom/notifications/send_scheduled_notifications.py

```diff
@@ -76,7 +76,7 @@ def run_schedules(self, force: bool):
                 self.process_schedule(schedule, user, company, now_utc, user_topic_map.get(user["_id"]) or {}, force)
             except Exception as e:
                 logger.exception(e)
-                logger.error(f"{self.log_msg} Failed to run schedule for user {user_id}")
+                logger.error("Failed to run schedule for user", extra={"user": user_id})
```

Comment:
This may work for sentry, but on console logs will no longer output the `user_id` (without changing the formatter)